src/formatDate.js

```diff
@@ -50,7 +50,30 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
-}
+  const currentDateOld = date.split(fromFormat[3]);
+  const res = [];
+  const dateInfo = {};
+
+  for (let i = 0; i < 3; i++) {
```

Comment:
It is better not use "magic" numbers as `3`. It's hard to understand what does it mean. 
I would recommend to use additional constants for these numbers
Ex: 
`const MY_VARIABLE = 3`